### ISSUE TITLE
Publish WCAG 2.2 updates to the Design System website

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#3405]
## What
Following completion of the [comms plan](https://github.com/alphagov/govuk-design-system/issues/3310), we will be ready to publish updates to the website on 11 Jan.

## Why
To help users understand how WCAG 2.2 impacts Design System components and patterns.

## Who needs to work on this
Calvin, Charlotte

## Who needs to review this
Steve, Kelly, Accessibility blog owner, GDS social media owner

## Done when
*11 Jan*
- [ ] we have announced the updates to the mailing list
- [ ] we have announced the updates on Slack
- [ ] messages have been posted on GDS Twitter and LinkedIn
- [ ] the 'What's new' section on the website has been updated
- [ ] "Get to WCAG 2.2 faster with the GOV.UK Design System" launch blog has been published
- [ ] we have set up a GitHub discussions space for WCAG 2.2

*w/c 15 Jan*
- [ ] we have sent decision making rationale to design sprint research participants and workshop attendees

**Comment by CharlotteDowns** [comment]
[GitHub discussions space for WCAG 2.2](https://github.com/alphagov/govuk-design-system/discussions/categories/wcag-2-2)

**Comment by calvin-lau-sig7** [comment]
Publish order:

1. Content
2. Blog
3. Comms
